### DISCUSSION TITLE
Using Text Tokens In an image node?

### DISCUSSION THREAD

**Discussion by WinstonWoof** [WASasquatch/was-node-suite-comfyui#115] [Q&A] [answered]
This might be me not seeing the wood for the trees but as per the subject " how do you use Text Tokens in Image Nodes please?

**Comment by WASasquatch** [comment]
You mean in inputs of image handling nodes? I am a little confused.

**Reply by WinstonWoof** [reply]
Hi

https://github.com/WASasquatch/was-node-suite-comfyui#text-tokens

So for example if I put   **[hostname]_[time]__[time(%Y-%m-%d__%I-%M%p)]** in the filename box for an Image Output step (which is how I read the above link) the filename gets literally written as the above suffixed with the autoadded number by COmfy itself

**Reply by WASasquatch** [reply, marked as answer]
Are you using WAS-NS Image Save node? `WAS Suite -> IO -> Image Save`

**Reply by WinstonWoof** [reply]
/me slaps forehead

FFS why didn't I think of looking to see if there was a custom Image Save option that was being referred too instead of making the dumbass presumption it was the Standard Image Save option

**Reply by WASasquatch** [reply]
It would be nice to be able to patch into base nodes. But that may lead to confliction if others do as well.

**Reply by WinstonWoof** [reply]
it makes sense but it was just the intuitive (to me at least) thing to do
